network-store-iidm-impl/src/test/java/com/powsybl/network/store/iidm/impl/VoltageLevelSetVAngleInCalculatedViewsTest.java

```diff
@@ -0,0 +1,667 @@
+/**
```

Comment:
Should this file be here or in powsybl-core TCK?

Reply:
maybe move it in a later PR ?